File: app/src/main/java/jgappsandgames/smartreminderslite/date/MonthAdapter.java
package jgappsandgames.smartreminderslite.date;

// Java
import java.util.ArrayList;

// App
import jgappsandgames.smartreminderslite.adapter.TaskAdapterInterface;

// Save
import jgappsandgames.smartreminderssave.tasks.Task;

/**
 * MonthAdapter
 * Created by joshua on 10/14/17.
 */
class MonthAdapter extends TaskAdapterInterface {
    MonthAdapter(MonthActivity activity, ArrayList<Task> tasks) {
        super(activity, activity, tasks);
    }
}

File: app/src/main/java/jgappsandgames/smartreminderslite/sort/date/MonthAdapter.kt
package jgappsandgames.smartreminderslite.sort.date

// Java
import java.util.ArrayList

// Save
import jgappsandgames.smartreminderssave.tasks.Task

// App
import jgappsandgames.smartreminderslite.adapter.TaskAdapterInterface

/**
 * MonthAdapter
 * Created by joshua on 1/19/2018.
 */
class MonthAdapter(activity: MonthActivity, tasks: ArrayList<Task>): TaskAdapterInterface(activity, activity, tasks)